packages/matter-node.js/src/examples/BridgedDevicesNode.ts

```diff
@@ -97,7 +97,9 @@ class BridgedDevice {
 
         const matterServer = new MatterServer(storageManager, netAnnounceInterface);
 
-        const uniqueId = Time.nowMs(); // TODO Store it!
+        //console.log("UNID: " + getIntParameter("unid"));
+
+        const uniqueId = getIntParameter("unid"); //Time.nowMs(); // TODO Store it!
```

Comment:
Good idea. Please also name parameter uniqueid and yes ideally put into storage and use like done with other parameters

Reply:
sure